ChangeLog

```diff
@@ -1,5 +1,6 @@
 * 0.10.0
-  * remove `DictionaryUtil`
```

Comment:
"Removed" and "Added"
http://keepachangelog.com/en/0.3.0/ (also http://keepachangelog.com/ru/0.3.0/)